packages/veritone-widgets/src/widgets/FolderSelectionDialog/styles.scss

```diff
@@ -0,0 +1,181 @@
+  left:50%;
+  top:50%;
+  transform:translate(-50%,-50%);
+  position:fixed;
+  z-index: 9999;
+  min-height: 80px;
+}
+
+.selected {
+  background-color: #E8F0FE;
+  border-top-right-radius: 50px;
+  border-bottom-right-radius: 50px;
+}
+
+.rootfolder {
+  &:hover {
+    background-color: #EEE;
+    border-top-right-radius: 50px;
+    border-bottom-right-radius: 50px;
+  }
+}
+
+.error {
+  color: #F44236;
+}
+
+.dialogTitle {
+  padding: 24px 24px 5px !important;
+}
+
+.dialogSubTitle {
+  padding: 0 24px 20px !important;
+}
+
+.dialogContent {
+  border: 2px solid #eee;
+  margin-right: 20px;
+  margin-left: 20px;
+  padding: 0 !important;
+}
+
+.closeIcon {
+  font-size: 30px !important;
+  margin-right: 20px;
+  margin-top: 20px;
+  color: #515154;
+}
+
+.workIcon {
+  font-size: 30px !important;
+  margin: 10px;
+  color: #515154;
+}
+
+.saveText {
+  color: #1873E8 !important;
+  padding-left: 30px !important;
+  padding-top: 5px !important;
+  padding-bottom: 5px !important;
+}
+
+.button {
+  min-height: 50px !important;
+}
+
+.buttonNewFolder {
+  border: 1px solid #1873E8 !important;
+}
+
+.buttonSelect {
+  background-color: #1873E8 !important;
+  margin-left: 5px !important;
+  &:hover {
+    background-color: #2195F3 !important;
+  }
+}
+
+
+
+.newFolderButton {
+  color: #1873E8 !important;
+  font-size: 18px !important;
+  font-weight: bold !important;
+  text-transform: none !important;
+}
+
+.cancelButton {
+  font-size: 18px !important;
+  font-weight: bold !important;
+  text-transform: none !important;
+  color: #515154 !important;
+}
+
+.selectButton {
+  font-size: 18px !important;
+  font-weight: bold !important;
+  text-transform: none !important;
+  color: #fff !important;
+}
+
+.folder {
+  height: 50px;
+  display: flex;
+  align-items: center;
+  color: #515154;
+}
+
+.collapsibleFolderIcon {
+  font-size: 30px !important;
+  color: #515154;
+}
+
+.folderIcon {
+  font-size: 30px !important;
+  color: #515154 !important;
+  margin-left: 30px;
+}
+
+.folderName {
+  font-weight: 600;
+  color: #515154;
+  padding-left: 10px;
+}
+
+.folderNameSelected {
+  color: #1873E8;
+}
+
+
+
+.folderPicker ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  overflow: hidden;
+}
+
+.folderPicker li {
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+}
+
+.folderPicker li > div:hover {
+  background-color: #EEE;
+  border-top-right-radius: 50px;
+  border-bottom-right-radius: 50px;
+}
+
+.folderPicker li div:first-child  {
+  padding-left: 24px;
+  overflow: hidden;
+}
+
+.folderPicker li li div:first-child  {
+  padding-left: 48px;
+  overflow: hidden;
+}
+
+.folderPicker li li li div:first-child  {
+  padding-left: 72px;
+  overflow: hidden;
+}
+
+.folderPicker li li li li div:first-child  {
+  padding-left: 96px;
+  overflow: hidden;
+}
+
+.folderPicker li li li li li div:first-child  {
+  padding-left: 120px;
+  overflow: hidden;
+}
+
+.folderPicker li li li li li li div:first-child  {
+  padding-left: 144px
+}
+
```

Comment:
Are we supporting max 6 levels?

What if we have 7?

Reply:
there can only be a depth of 5 levels from the root per graphql backend - added another as safety - in case they change it